### PR TITLE
PR for issue #4915 "fix ipywidgets"

### PR DESCRIPTION
**Pull request by konstantinTarletski**

### CODE REVIEW

File: core/src/main/java/com/twosigma/beaker/core/rest/PluginServiceLocatorRest.java [hunk 1]
```diff
@@ -818,10 +818,20 @@ public String getIPythonVersion(@QueryParam("pluginId") String pluginId,
     proc = Runtime.getRuntime().exec(listToArray(cmd), buildEnv(pluginId, null));
     BufferedReader br = new BufferedReader(new InputStreamReader(proc.getInputStream()));
     new StreamGobbler(proc.getErrorStream(), "stderr", "ipython-version", null, null).start();
-    String line = br.readLine();
-    return line;
+
```
Comment (scottdraves) [inline core/src/main/java/com/twosigma/beaker/core/rest/PluginServiceLocatorRest.java]: what are these changes about?

File: plugin/ipythonPlugins/src/dist/python3/python3Plugin [hunk 2]
```diff
@@ -21,7 +21,7 @@ import subprocess
 if len(sys.argv) > 1:
     arg1 = sys.argv[1]
     if arg1 == "--version":
-        os.execlp("ipython", "ipython", "--version")
+        os.execlp("ipython", "ipython", "-c", "\"import pip; import IPython; print( \'ipywidgets==\' + \'%s\' % sorted(filter(lambda x : x.key == \'ipywidgets\', pip.get_installed_distributions()))[0].version ) ; print(\'ipython==\' + str(IPython.version_info[0])  + '.' + str(IPython.version_info[1]) + '.' + str(IPython.version_info[2]))\"");
```
Comment (scottdraves) [inline plugin/ipythonPlugins/src/dist/python3/python3Plugin]: why not just `ipywidgets.__version__`?

Comment (konstantinTarletski) [reply plugin/ipythonPlugins/src/dist/python3/python3Plugin]: I was not succeeded in "ipywidgets.__version__"
This code is not last version, 
Last one like :

arr = list(filter(lambda x : x.key == \'ipywidgets\', pip.get_installed_distributions())); print (\'ipywidgets==%s\' % arr[0].version) if len(arr) > 0 else \'NONE\';

**Comment by EfimovVladimir** [comment]
kick jenkins

**Comment by EfimovVladimir** [comment]
kick jenkins

**Comment by EfimovVladimir** [comment]
kick jenkins

**Comment by EfimovVladimir** [comment]
kick jenkins

**Pull request merged**